release/ray_release/retry.py

```diff
@@ -0,0 +1,38 @@
+"""Utils on retry."""
+
+import time
+from functools import wraps
+
+_RAY_DEFAULT_MAX_RETRY_COUNT = 10
+_RAY_DEFAULT_INIT_DELAY_SEC = 1
+_RAY_DEFAULT_MAX_DELAY_SEC = 30
+_RAY_DEFAULT_BACKOFF = 2
+_RAT_DEFAULT_JITTER_SEC = 1
```

Comment:
maybe drop the `_RAY` prefix. it is private (to this file) so you can just say `_DEFAULT` directly.

Reply:
Sure, updated.